readme.md

```diff
@@ -1,6 +1,6 @@
 # Stopwatch task
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://OleksandrFEdev /layout_stop-watch.github.io/layout_stop-watch/)
```

Comment:
fix the demo link to https://YOUR_GITHUB_ACCOUNT_NAME.github.io/DEPLOYED_FOLDER_NAME

Reply:
- [DEMO LINK](https://OleksandrFEdev.github.io/layout_stop-watch/)